CHANGELOG.md

```diff
@@ -1,3 +1,155 @@
+# 84.4.0
+
+In this minor release we added additional dynamic type support for Buttons, Chips, and List, as well as additional tests and bug fixes.
+
+## New deprecations
+
+`mdc_legacyFontScaling` in `MDCButton` and `MDCChipView` has been deprecated. 
+Please use `adjustsFontForContentSizeCategoryWhenScaledFontIsUnavailable` instead.
+
+## New features
+
+Buttons, Chips, and List, now support dynamic type when used with an MDCFontScaler:
```

Comment:
Please avoid any statements that imply that we did not support Dynamic Type before. Consider rewording this to: "Buttons, Chips, and List, have expanded their support for Dynamic Type to allow custom scaling curves to be associated with fonts".